Packages/cc.starlessnight.unity-jsb/Source/Binding/Values_match.cs

```diff
@@ -85,7 +85,7 @@ public static bool js_match_type(JSContext ctx, JSValue jsValue, Type type)
                     var types = context.GetTypeDB();
                     var o = types.GetType(type_id);
                     // UnityEngine.Debug.Log($"get type from exported registry {o}:{type_id} expected:{type}");
-                    return o != null && (o == type || o.IsSubclassOf(type));
+                    return o != null && (o == type || type.IsAssignableFrom(o));

```

Comment:
When a method has a parameter typed with an interface, it won't work if using `IsSubclassOf`.